src/test/java/Test/LoginTest.java

```diff
@@ -8,5 +8,6 @@ public class LoginTest extends BaseClass {
     public void LoginTest() throws InterruptedException {
```

Comment:
Why we are over writing existing test we can add a new class and write the new Test there